content/hardware/05.pro-solutions/solutions-and-kits/portenta-proto-kit-ve/datasheet/datasheet.md

```diff
@@ -0,0 +1,380 @@
+
+<div style="page-break-after: always;"></div>
+
+## Mechanical Information
+
+<p style="text-align: justify;">
+The Arduino Portenta Proto Kit VE offers significant mechanical flexibility, supporting multiple configurations based on the combination of components used. This section provides the main dimensions of one possible configuration for reference. For detailed mechanical specifications of each individual component, please consult the corresponding documentation available on Arduino Docs.
+</p>
+
+### Kit Dimensions
+
+<p style="text-align: justify;">
+The figures below show the main dimensions of the kit in a stacked configuration that includes the Portenta H7 board, the Portenta Mid Carrier, the Portenta Mid Carrier Proto Shield, the Arduino Pro 4G Module, one Modulino (Pixels) and the Nicla Vision board. While the Nicla Sense Env is part of the kit, it is not physically stacked with these components. Instead, it connects to the system via an ESLOV cable, whose length may vary depending on the user's setup. As a result, the Nicla Sense Env and its connection are not represented in this stacked configuration. All dimensions are in millimeters (mm).
+</p>
+
+- <p style="text-align: justify;"><strong>Top View</strong>: Displays the width and length of the stacked components configuration of the kit.</p>
+
+![](assets/proto-kit-example-mechanical-1.png)
+
+<div style="page-break-after: always;"></div>
+
+- <p style="text-align: justify;"><strong>Side View</strong>: Displays the height of the stacked components configuration of the kit.</p>
+
+![](assets/proto-kit-example-mechanical-2.png)
+
+<div style="page-break-after: always;"></div>
+
+## Product Compliance
+
+<p style="text-align: justify;">The Arduino Portenta Proto Kit VE consists of multiple individual Arduino products, each of which complies with specific regulations and certifications. For detailed product compliance information, please refer to the corresponding datasheets of each component included in the kit:</p>
+
+- [Portenta H7 Documentation](https://docs.arduino.cc/hardware/portenta-h7/) **[8]**
+- [Nicla Vision Documentation](https://docs.arduino.cc/hardware/nicla-vision/) **[9]**
+- [Nicla Sense Env Documentation](https://docs.arduino.cc/hardware/nicla-sense-env/) **[10]**
+- [Portenta Mid Carrier Documentation](https://docs.arduino.cc/hardware/portenta-mid-carrier/) **[11]**
+- [Arduino Pro 4G Module Documentation](https://docs.arduino.cc/hardware/pro-4g-module/) **[12]**
+
```

Comment:
Portenta Mid Carrier Proto Shield will need to be added here as well, as part of the list

Reply:
It was not added because no page on Arduino Docs is dedicated to the Proto Shield.